### DISCUSSION TITLE
Choose a Snowflake role for transfer

### DISCUSSION THREAD

**Discussion by rbayliss** [sqlpipe/sqlpipe#12] [Q&A]
Hello! 
We're evaluating this tool, and can't seem to find a way to set the role for the Snowflake session. Is it possible to set the role to use, or has this not been implemented yet? Thanks!

**Comment by calmitchell617** [comment]
Hey Rob, that is not implemented yet, and I'm not doing any dev work on SQLpipe unless it's sponsored right now. I have a few paying clients that take my time.
